kedro/framework/project/__init__.py

```diff
@@ -270,6 +276,32 @@ def validate_settings():
     importlib.import_module(f"{PACKAGE_NAME}.settings")
 
 
+def _create_pipeline(pipeline_module: Optional[types.ModuleType]) -> Optional[Pipeline]:
+    if pipeline_module is None:
+        return None
```

Comment:
Is this case still relevant? Does `_create_pipeline` ever get called with `pipeline_module = None` after the changes you made yesterday?

Reply:
Shouldn't happen! Let me remove this and update the function signature.